hack/devtools/local_dev_env.sh

```diff
@@ -11,7 +10,7 @@ set -o pipefail
 
 PLATFORM_WORKLOAD_IDENTITY_ROLE_SETS='[
     {
-        "openShiftVersion": "4.14",
+        "openShiftVersion": "4.15",
```

Comment:
I would suggest populating this file with the full 4.14 and 4.15 role sets in case someone sources this script and wants to create a 4.14 cluster instead.

Reply:
Good idea. I'll just need to make sure that all scripts are able to select the correct roleset based on the version. Right now, i think we only ever expect a single roleset to be present.